src/components/Dashboard/FilingHistoryList/subtitles/SubtitleFutureEffectivePaid.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div class="subtitle-future-effective-paid">
     <span v-if="isTypeIncorporationApplication">FUTURE EFFECTIVE INCORPORATION</span>
+    <span v-else-if="isTypeContinuationApplication">FUTURE EFFECTIVE</span>
```

Comment:
Is there a UI design for this? I'm surprised it doesn't say "FUTURE EFFECTIVE CONTINUATION". If there's no design, check with Jacqueline.

Reply:
Yeah, this is what Jacqueline sent me:
![image](https://github.com/user-attachments/assets/9eca9487-10ea-442e-a4ee-84da54ccea65)
The future effective text and contact info were not in the original design (
https://www.figma.com/design/rNe2jn8wyFykKRzwiBo6ar/Continuation-In-V6?node-id=100-4595&m=dev&focus-id=664-72490
- design is at the bottom)

Reply:
Excellent. Thanks. Maybe add that Figma link to the ticket so QA can find it, too.

PS I really like the new look with the dividers :)